driver/registry_default_login.go

```diff
@@ -29,6 +29,16 @@ func (m *RegistryDefault) PostLoginHooks(ctx context.Context, credentialsType id
 			b = append(b, hook)
 		}
 	}
+
+	if len(b) == 0 {
+		// since we don't want merging hooks defined in a specific strategy and global hooks
+		// global hooks are added only if no strategy specific hooks are defined
+		for _, v := range m.getHooks("global", m.Config(ctx).SelfServiceFlowLoginAfterHooks("global")) {
```

Comment:
Replace global with https://github.com/ory/kratos/pull/1240#discussion_r619988905

Reply:
This is only in code to differentiate between strategy and "global" hooks. In the config the "hook" property can be configured directly within the "after" property.

Reply:
Sorry about that confusion!